sycl/source/detail/context_impl.hpp

```diff
@@ -167,6 +167,9 @@ class context_impl {
   /// \return a native handle.
   pi_native_handle getNative() const;
 
+  // Returns true if buffer_location property is supported by devices
+  bool isBufferLocationSupported(); 
```

Comment:
```suggestion
  bool isBufferLocationSupported() const; 
```

Can we have it's implementation in header so that compiler can inline a function at caller's side if possible?

Reply:
This function is not simple enough to be in header